client/src/Components/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import "./CustomButton.scss";
+
+const CustomButton = ({ children, ...otherProps }) => {
+  return (
+    <button className={"custom-button"} {...otherProps}>
+      {children}
+    </button>
+  );
+};
+
+export default CustomButton;
```

Comment:
good job. Try to think creating more such individual atomic structures like Link, Image etc